Makefile

```diff
@@ -18,8 +26,28 @@ FFMPEG_OPTS ?= --prefix=$(FFMPEG_BUILD) \
                --enable-parser=h264 \
                --enable-parser=hevc
 
+CGO_LDFLAGS := -L$(FFMPEG_PREFIX)/lib
+
+ifeq ($(TARGET_S),android)
+	export CGO_ENABLED = 1
+	NDK_ROOT ?= $(HOME)/Library/Android/Sdk/ndk/26.1.10909125
+	export CC = $(NDK_ROOT)/toolchains/llvm/prebuilt/$(UNAME_S)-x86_64/bin/aarch64-linux-android$(API_LEVEL)-clang
```

Comment:
1. What are `x86_64` and `aarch64` are hard coded on all source platforms? I don't understand why they don't need to change depending on source & target os & arch.

Reply:
Yeah I am a little confused as well... For some reason on darwin/arm64 source uses x86_64 path. I can try this out on x86/linux source as well to figure out how we want to handle this.

Reply:
Can you please document what you find in comments?